src/Twilio/Twilio.csproj

```diff
@@ -1,15 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup Label="NuGet">
-    <TargetFrameworks>netstandard1.4;netstandard2.0;net451;net35</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net35</TargetFrameworks>
```

Comment:
net35 and netstandard2.0 will support all versions of .NET that Microsoft supports, .NET Framework 3.5 and 4.6.2+, and .NET 6+.